v2/exchange_info_service.go

```diff
@@ -29,6 +30,16 @@ func (s *ExchangeInfoService) Symbols(symbols ...string) *ExchangeInfoService {
 	return s
 }
 
+// Permissions set permission
+func (s *ExchangeInfoService) Permissions(permissions ...string) *ExchangeInfoService {
+	if len(permissions) == 0 {
+		s.permissions = "[]"
+	} else {
+		s.permissions = "[\"" + strings.Join(permissions, "\",\"") + "\"]"
```

Comment:
I don't think it's a good idea to hard code json string, it's better to use `[]string` instead of `string` for `permissions`

Reply:
yes, i think so, i using this approach based on previous implementation for `symbols` param. but if we use `[]string` we should create adjustment in setParams method. with the current code using `[]string` will make the code not work.

Reply:
> yes, i think so, i using this approach based on previous implementation for `symbols` param. but if we use `[]string` we should create adjustment in setParams method. with the current code using `[]string` will make the code not work.

Could you modify `setParam` method as well to process the vault of type `[]string`? It seems like an easy change.

Reply:
already make changes on `setParam` method. please check